examples/cookbook/api-keys/near-connection.js

```diff
@@ -0,0 +1,29 @@
+// demonstrates how to use API-KEY with 'connect' function. 
+const { connect, keyStores } = require("near-api-js");
+const path = require("path");
+
+const homedir = require("os").homedir();
+const CREDENTIALS_DIR = ".near-credentials";
+const credentialsPath = path.join(homedir, CREDENTIALS_DIR);
+const keyStore = new keyStores.UnencryptedFileSystemKeyStore(credentialsPath);
+
+const RPC_API_ENDPOINT = 'http://34.141.205.230:3030/';
+const API_KEY = 'ZmowOTMyNHUwMnUzNDA5MnUzMDk0dTIzeA==';
```

Comment:
I believe API keys are going to be in UUID v4 format, so giving an example with that might make it clearer for users. This looks like it also might be Base64 encoded, will users be required to do that?

Reply:
If we haven't settled on a final format for API keys, I think it would be pretty nice if our API keys were formatted in such a way that they don't need URL escaping when provided as an URL query string. What do you folks think?

Reply:
@mpeterdev I'm not sure in what format API Keys will be distributed. @chefsale what is the design for it? Is it finalized?
We can encode it to `base64` ourselves if needed.
@MaximusHaximus agree, I'm in for that.

Reply:
Just to close this discussion:
- UUID v4 is the final format
- base64 encoding is not required